### ISSUE TITLE
Can not make flex-cloud images -- still using MySQLdb

### ISSUE THREAD

**Issue by briandrawert** [StochSS/stochss#422]
```
brian@laptop:flex-cloud$ ./make_flex_vms.py -h
Traceback (most recent call last):
  File "./make_flex_vms.py", line 23, in <module>
    import MySQLdb
ImportError: No module named MySQLdb
```

**Comment by dnath** [comment]
Fixed in 1d7c4a1b4181e67bffa088003e809829d0ceb202

**Comment by briandrawert** [comment]
Closed in https://github.com/StochSS/stochss/pull/383
